data/RedHat.yaml

```diff
@@ -3,3 +3,4 @@ chrony::config_keys_group: chrony
 chrony::config_keys: /etc/chrony.keys
 chrony::config: /etc/chrony.conf
 chrony::wait_manage: true
+chrony::leapsectz: right/UTC
```

Comment:
That's a suspect

Reply:
http://kenta.blogspot.com/2016/03/sqfzcxay-understanding-right-time-zone.html

The value `right/UTC` is what RHEL ships with.